### ISSUE TITLE
Message "No shipping methods available for selected location, please change your address and try again." appeared trying to complete checkout

### ISSUE THREAD

**Issue by pmackay** [openfoodfoundation/openfoodnetwork#294]
Was able to get around this by creating a new Shipping method, but need to investigate more what the difference was.

**Comment by lin-d-hop** [comment]
Has this been seen again? Ive never seen this or heard it reported... such an old bug. 
Can I close @pmackay?

**Comment by pmackay** [comment]
Good idea.

**Comment by nanoharsha** [comment]
We have been facing this issue, not sure what is the logic/reason for the error. 
I tried creating new shipping method, changing shipping methods/modifying location of the order being placed, but doesn't solve the problem !!
Any one else has faced the issue since ?

**Comment by mkllnk** [comment]
We haven't come across this issue. It sounds like some kind of data inconsistency. Here are two more solutions: http://stackoverflow.com/questions/16051605/spree-no-shipping-methods-available-for-selected-location-please-change-your-a

**Comment by nanoharsha** [comment]
Thanks @mkllnk , went through the links and started going through configuration from Admin panel.
I tried fiddling with the zones a lil. I had country based zone, and didn't have states added.
I went back and added states also in the zones, and voila the orders went through fine !!
Do point out if there is a document floating about essential changes to be done in the configuration.
I will try to see if I can put some energies !! 

Thanks once again !!

**Comment by kirstenalarsen** [comment]
Hey Harsha - there is a page here - https://community.openfoodnetwork.org/t/initial-set-up-and-configuration/198 <https://community.openfoodnetwork.org/t/initial-set-up-and-configuration/198> - that has probably not been updated for quite a while so by all means send some energies there!!

**Comment by popok75** [comment]
Hi, I also got this bug, after few hours trying, for me it was due to the fact that I didn't add the country to the europe zone, and the state to the country zone.  So if you have this bug "edit" your zones, and check the states/countries displayed.

I didn't even notice the "add state/county" button when creating a new zone. Also I thought that the state was added to the country zone when I created it, I didn't know I have to add it also in the country zone.
